src/Microsoft.Extensions.FileSystemGlobbing/Abstractions/DirectoryInfoBase.cs

```diff
@@ -2,16 +2,32 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System.Collections.Generic;
-using System.IO;
 
 namespace Microsoft.Extensions.FileSystemGlobbing.Abstractions
 {
+    /// <summary>
+    ///     Represents a directory
```

Comment:
>  hard to maintain 

Not too worried about a rush of people making xmldocs changes...this code was written months ago but never got documented. :trollface: 

But I see your point. I've aligned to the VS indenting defaults.